### DISCUSSION TITLE
Downloaded README.md files don't render correctly in ANY markdown viewer

### DISCUSSION THREAD

**Discussion by Dan-Essin** [community/community#63304] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

There are a couple of readme files that I downloaded for future reference. I have tried viewing them with MarkdownPad, Markdown View, Typora, and GhostWriter. They all look the same, but not what they look like on GitHub.

On GitHub it looks like this:
**### Android for Python Users**

_An unofficial Buildozer Users' Guide_

Revised 2023-07-25
**### Table of Contents**

After downloading the file and opening it in a markdown editor it looks like this:
{“payload”:{“allShortcutsEnabled”:false,“fileTree”:{””:{“items”:[{“name”:“images”,“path”:“images”,“contentType”:“directory”},{“name”:“README.md”,“path”:“README.md”,“contentType”:“file”}],“totalCount”:2}},“fileTreeProcessingTime”:2.449862,“foldersToFetch”:[],“reducedMotionEnabled”:null,“repo”:{“id”:377338837,“defaultBranch”:“main”,“name”:“Android-for-Python-Users”,“ownerLogin”:“Android-for-Python”,“currentUserCanPush”:false,“isFork”:false,“isEmpty”:false,“createdAt”:“2021-06-16T01:39:01.000Z”,“ownerAvatar”:“https://avatars.githubusercontent.com/u/85974252?v=4","public":true,"private":false,"isOrgOwned":false},"refInfo":{"name":"main","listCacheKey":"v0:1623807542.294861","canEdit":false,"refType":"branch","currentOid":"efa2c9dc62470aed8ab8fda2e8921bd071aa3f74"},"path":"README.md","currentUser":null,"blob":{"rawLines":null,"stylingDirectives":null,"csv":null,"csvError":null,"dependabotInfo":{"showConfigurationBanner":false,"configFilePath":null,"networkDependabotPath":"/Android-for-Python/Android-for-Python-Users/network/updates","dismissConfigurationNoticePath":"/settings/dismiss-notice/dependabot_configuration_notice","configurationNoticeDismissed":null,"repoAlertsPath":"/Android-for-Python/Android-for-Python-Users/security/dependabot","repoSecurityAndAnalysisPath":"/Android-for-Python/Android-for-Python-Users/settings/security_analysis","repoOwnerIsOrg":false,"currentUserCanAdminRepo":false},"displayName":"README.md","displayUrl":"https://github.com/Android-for-Python/Android-for-Python-Users/blob/main/README.md?raw=true","headerInfo":{"blobSize":"117 KB”,“deleteInfo”:{“deleteTooltip”:“You must be signed in to make or propose changes”},“editInfo”:{“editTooltip”:“You must be signed in to make or propose changes”},“ghDesktopPath”:“https://desktop.github.com","gitLfsPath":null,"onBranch":true,"shortPath":"f2b3076","siteNavLoginPath":"/login?return_to=https%3A%2F%2Fgithub.com%2FAndroid-for-Python%2FAndroid-for-Python-Users%2Fblob%2Fmain%2FREADME.md","isCSV":false,"isRichtext":true,"toc":[{"level":1,"text":"Android for Python Users”,“anchor”:“android-for-python-users”,“htmlText”:“Android for Python Users”},{“level”:1,“text”:“Table of Contents”,“anchor”:“table-of-contents”,“htmlText”:“Table of Contents”},{“level”:1,“text”:“Introduction”,“anchor”:“introduction”,“htmlText”:“Introduction”},{“level”:1,“text”:“Confidence and Competence”,“anchor”:“confidence-and-competence”,“htmlText”:“Confidence and Competence”},{“level”:1,“text”:“What is Different about Android?”,“anchor”:“what-is-different-about-android”,“htmlText”:“What is Different about Android?”},

These are claimed to be markdown files but it seems that they must be some proprietary github format that has a .md extension.

What am I missing. Is there some way to convert these files to real markdown files? Or what?
I'm confused.

Thank you

**Comment by ayatakinci** [comment, marked as answer]
It seems like the files you downloaded are not actual Markdown files, but rather some GitHub-specific representation of the content. GitHub's interface provides a rendered view of Markdown files, but the raw content you downloaded seems to include metadata and formatting specific to GitHub's platform.
To convert these files into real Markdown files that you can view and edit with standard Markdown editors, you'll need to extract the actual Markdown content from the GitHub-specific payload. 
Download the Raw Content: Instead of downloading the file through GitHub's interface, try downloading the raw content of the Markdown file. To do this, you can navigate to the file on GitHub, click on the "Download" button (usually denoted by a "Raw" option), and save the file to your computer. 

Now open the downloaded file in a text editor, and manually remove any metadata, formatting, or additional JSON content that is not part of the actual Markdown content. You want to extract just the Markdown text.
Once you've extracted the Markdown content, validate it to ensure that it adheres to proper Markdown syntax. You can use online Markdown validators or parsers to check the validity of the Markdown.

and then after you've cleaned up the Markdown content, you can open it in a Markdown editor of your choice (such as Typora, VS Code with appropriate extensions, etc.) to view and edit it.

If the raw content still doesn't look like standard Markdown or if you're having trouble extracting it, you may want to consider reaching out to the repository owner or maintainer for guidance on how to access the actual Markdown content. They might provide you with a downloadable link to the raw Markdown file.
